### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by GitHubPang**

### CODE REVIEW

**Review by Rick-Anderson — APPROVED** [review]

**Comment by Rick-Anderson** [comment]
https://review.docs.microsoft.com/en-us/aspnet/core/blazor/test?view=aspnetcore-3.1&branch=pr-en-us-23098

File: aspnetcore/signalr/configuration.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ var connection = new HubConnectionBuilder()
 
 ### Switch to Newtonsoft.Json
 
-If you need features of `Newtonsoft.Json` that aren't supported in `System.Text.Json`, See [Switch to Newtonsoft.Json](xref:migration/22-to-30#switch-to-newtonsoftjson).
+If you need features of `Newtonsoft.Json` that aren't supported in `System.Text.Json`, see [Switch to Newtonsoft.Json](xref:migration/22-to-30#switch-to-newtonsoftjson).
```
Comment (guardrex) [inline aspnetcore/signalr/configuration.md:51]:
```suggestion
If you need features of `Newtonsoft.Json` that aren't supported in `System.Text.Json`, see [Switch to `Newtonsoft.Json`](xref:migration/22-to-30#switch-to-newtonsoftjson).
```

File: aspnetcore/signalr/configuration.md [hunk 2]
```diff
@@ -442,7 +442,7 @@ var connection = new HubConnectionBuilder()
...
```
Comment (guardrex) [inline aspnetcore/signalr/configuration.md:445]:
```suggestion
If you need features of `Newtonsoft.Json` that aren't supported in `System.Text.Json`, see [Switch to `Newtonsoft.Json`](xref:migration/22-to-30#switch-to-newtonsoftjson).
```

File: aspnetcore/signalr/configuration.md [hunk 3]
```diff
@@ -835,7 +835,7 @@ var connection = new HubConnectionBuilder()
...
```
Comment (guardrex) [inline aspnetcore/signalr/configuration.md:838]:
```suggestion
If you need features of `Newtonsoft.Json` that aren't supported in `System.Text.Json`, see [Switch to `Newtonsoft.Json`](xref:migration/22-to-30#switch-to-newtonsoftjson).
```

File: aspnetcore/signalr/configuration.md [hunk 4]
```diff
@@ -1222,7 +1222,7 @@ var connection = new HubConnectionBuilder()
...
```
Comment (guardrex) [inline aspnetcore/signalr/configuration.md:1225]:
```suggestion
If you need features of `Newtonsoft.Json` that aren't supported in `System.Text.Json`, see [Switch to `Newtonsoft.Json`](xref:migration/22-to-30#switch-to-newtonsoftjson).
```

**Review by guardrex — APPROVED** [review]
Approving with a tiny nit to fence the package name for localization.

**Comment by Rick-Anderson** [comment]
@GitHubPang  thanks for the PR.

**Pull request merged**